packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableDetailsSteps.tsx

```diff
@@ -96,16 +104,22 @@ const Step = ({
     <div
       className={twMerge(
         'my-3 flex h-3 items-center justify-between space-x-2',
-        pending && 'animate-pulse'
+        pending && 'animate-pulse',
+        extendHeight && 'h-auto items-start'
```

Comment:
I tried your suggestion. It's okay for the height, but we would still need the conditional for `extendHeight` because otherwise it messes with the flex layout setting of the div. So i feel it's better to leave it as is.

Reply:
sure